cli/command/image/load.go

```diff
@@ -34,12 +37,14 @@ func NewLoadCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags.StringVarP(&opts.input, "input", "i", "", "Read from tar archive file, instead of STDIN")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress the load output")
+	flags.BoolVarP(&opts.oci, "oci", "", false, "Load from an OCI image layout tar archive")
```

Comment:
we probably won't need it but as explained in the first comment load isn't finished yet - I was hoping to have a _go ahead_ to implement it if you maintainers agreed on the design here.